frontend/src/views/omni/Overview/Overview.vue

```diff
@@ -170,6 +173,12 @@ const hasRoleNone = computed(() => {
   return !role || role === RoleNone;
 });
 
+const hasRoleAdmin = computed(() => {
+  const role = currentUser.value?.spec?.role
```

Comment:
I'd suggest to create a separate flag like `canDownloadAuditLog` in the currentUser resource.

Reply:
Done!